### ISSUE TITLE
The security ownership document seems incorrect

### ISSUE THREAD

**Issue by videni** [oroinc/crm#308]
Since Mike only belongs to Second Organization, he is not assigned to Business Unit under Main Organization ,how could he create account in Main organization,   it seems 

> Imagine that each user created two accounts (one in Main Organization and one in Second Organization):

this precondition is not correct , am I right?  Both John and Mike are assigned to Child Business Unit ,  when John access level is Business Unit,    should John can also see Account G?  

![image](https://user-images.githubusercontent.com/4058341/41759912-7e03b6bc-7623-11e8-81a8-3ae84ae42c43.png)

https://oroinc.com/orocrm/doc/current/dev-guide/getting-started-book/security#examples

**Comment by yurio** [comment]
Hi, @videni.

Yes, in this schema, Mike does not able to create the data at the first organization. But imagine that all of this users had access to both organizations and to all Business units before, and at this moment they had create the data records.

John should not see Account G at Business unit access level because the creator of Account G (Mike) has no accessed business units from First organization that accessed to John.

In other words. Let's take John. He works at First organization and access level os Business unit.
At this First organization, John has assigned BU A. This Business unit also assigned to Mary and Robert. So, John should see Account A (his account), Account B (from Mary) and Account H (from Robert).

**Comment by videni** [comment]
@yurio , I see, thanks a lot.
